docs/guides/08-interactivity-events.md

```diff
@@ -0,0 +1,411 @@
+## Interactivity and events
+In this guide you will learn how to deal with user interactions within your visualization. After going through it, you will be able to manage some interesting events (e.g. waiting for a layer to load or clicking on a feature) to give your users more dynamic and useful visualizations. You will also learn how to build very common add-ons such as *pop-ups* and legends.
+
+At the end of the guide you will have built a visualization like this:
+<div class="example-map">
+    <iframe
+        id="guides-interactivity-step-final"
+        src="/developers/carto-vl/examples/maps/guides/interactivity/step-4.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+To start grab the source from a working template like this [basemap](/developers/carto-vl/examples/maps/guides/getting-started/step-1.html). Copy its source code into a new file called `interactivity.html` and test it is working fine before going on.
+
+Add a navigation control to the map, with:
+```js
+// Add zoom controls
+const nav = new mapboxgl.NavigationControl();
+map.addControl(nav, 'top-left');
+```
+
+### Map events
+It all begins with the map and sometimes you are interested in listening to some relevant events from to the **map** itself. For example you want to wait for it to load or maybe display the current map's center coordinates. In those cases, you can use a set of events already provided by the `Mapbox GL JS Map`, such as [load](https://www.mapbox.com/mapbox-gl-js/api#map.event:load) and [move](https://www.mapbox.com/mapbox-gl-js/api#map.event:move) respectively, and attach some callback functions to react on them.
```

Comment:
I would use bold style instead of code style in `Mapbox GL JS Map`

Reply:
", and attach ~~some~~ callback functions..."

Reply:
👍